Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,18 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const totalEarnings = tasks
+    .map(x => x.duration / 60)
+    .filter(x => x >= 2)
```

Comment:
Since the mapping operation mapped tasks to hours, the `x` of of the filter callback function is now an `hour`.